backend/src/handlers/outcomes_handler.go

```diff
@@ -28,11 +28,13 @@ func (srv *Server) HandleGetOutcomes(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 	}
 
-	// Get type from query param
+	// Get vars from query params
 	typeString := r.URL.Query().Get("type")
+	order := r.URL.Query().Get("order")
+	orderBy := r.URL.Query().Get("orderby")
```

Comment:
I would like to make all these standard across all the endpoints. Can we go with `order_by` ?